src/pages/workspace/expensifyCard/EmptyCardView.tsx

```diff
@@ -6,25 +6,30 @@ import ScrollView from '@components/ScrollView';
 import CardRowSkeleton from '@components/Skeletons/CardRowSkeleton';
 import Text from '@components/Text';
 import useLocalize from '@hooks/useLocalize';
+import useSafeAreaInsets from '@hooks/useSafeAreaInsets';
 import useThemeStyles from '@hooks/useThemeStyles';
 import useWindowDimensions from '@hooks/useWindowDimensions';
+import getPlatform from '@libs/getPlatform';
 import colors from '@styles/theme/colors';
 import CONST from '@src/CONST';
 
 const HEADER_HEIGHT = 80;
 const BUTTON_HEIGHT = 40;
 const BUTTON_MARGIN = 12;
 
+const isIOSNative = getPlatform() === CONST.PLATFORM.IOS;
+
 function EmptyCardView() {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
     const {windowHeight, isSmallScreenWidth} = useWindowDimensions();
 
-    const headerHeight = isSmallScreenWidth ? HEADER_HEIGHT + BUTTON_HEIGHT + BUTTON_MARGIN : HEADER_HEIGHT;
+    const safeAreaInsets = useSafeAreaInsets();
+    const headerHeight = isSmallScreenWidth ? HEADER_HEIGHT + BUTTON_HEIGHT + BUTTON_MARGIN + (isIOSNative ? safeAreaInsets.top : 0) : HEADER_HEIGHT;
```

Comment:
@koko57  We should create a function to get `headerHeight` and incorporating specific logic for each platform determined by the file name. See getPlaidLinkTokenParameters function as an example

Reply:
we're using getPlatform in ComposerWithSuggestions, so why not use it here? For ios it would need to be a hook, not a regular function or we would need to pass safeAreaInsets as a param to the function and would need to ignore it for web/android implementation? I think it's unnecessary complication as we have getPlatform

Reply:
@koko57 It isn't recommended in our codebase. Please see this guideline

https://github.com/Expensify/App/blob/3775daee3f0736add3ca1810123a5ff0e3f44a96/contributingGuides/PR_REVIEW_GUIDELINES.md?plain=1#L45

Reply:
cc @mountiny @MariaHCD (This PR belongs to Workspace Feed Project) Tag you guys to confirm

Reply:
@koko57 Any update here?

Reply:
@DylanDylann I don't see using getPlatform is discouraged, only a proper naming is mentioned. I think in this case we don't need platform specific function as it creates more unnecessary complexity.

curious wdyt @mountiny